test-openshift.yaml

```diff
@@ -68,4 +73,4 @@
 
         # Run tests on OpenShift 4
         - name: Run a sclorg test suite in OpenShift 4
-          shell: VERSION={{ environment[0]['VERSION'] }} IMAGE_NAME={{ image_name }} OS={{ environment[0]['OS'] }} CVP=1 bash test/run-openshift-remote-cluster  | tee {{ cvp_artifacts_dir }}/{{ image_name }}.log
+          shell: VERSION={{ environment[0]['VERSION'] }} IMAGE_NAME={{ image_name }} OS={{ environment[0]['OS'] }} CVP=1 bash ./run-openshift-remote-cluster  | tee {{ cvp_artifacts_dir }}/{{ image_name }}.log
```

Comment:
The `run-openshift-remote-cluster` is in `test` directory, why this change?

Reply:
I will clarify it, what is working directly

Reply:
Fixed by https://github.com/sclorg/container-common-scripts/pull/280/commits/0498bcb4329668ccbb026795e99315932021ff83